### DISCUSSION TITLE
[Documentation] Add "built with pydantic" section

### DISCUSSION THREAD

**Discussion by jhnnsrs** [pydantic/pydantic#4798] [Projects using Pydantic]
### Initial Checks

- [X] I have searched Google & GitHub for similar requests and couldn't find anything
- [X] I have read and followed [the docs](https://pydantic-docs.helpmanual.io) and still think this feature is missing

### Description

Out of my love for pydantic grew a little graphql-code generator that is able to generate pydantic models 
for graphql operations (inputtypes, types, fragments,..) allowing typesafe serialization between graphql and pydantic
models.

https://github.com/jhnnsrs/turms

As the datamodel-generator is referenced in the documentation, i was wondering if there are plans to add a section for
"built with pydantic" software like this and others?

Thanks for the great work!

### Affected Components

- [ ] [Compatibility between releases](https://pydantic-docs.helpmanual.io/changelog/)
- [ ] [Data validation/parsing](https://pydantic-docs.helpmanual.io/usage/models/#basic-model-usage)
- [X] [Data serialization](https://pydantic-docs.helpmanual.io/usage/exporting_models/) - `.dict()` and `.json()`
- [ ] [JSON Schema](https://pydantic-docs.helpmanual.io/usage/schema/)
- [ ] [Dataclasses](https://pydantic-docs.helpmanual.io/usage/dataclasses/)
- [ ] [Model Config](https://pydantic-docs.helpmanual.io/usage/model_config/)
- [ ] [Field Types](https://pydantic-docs.helpmanual.io/usage/types/) - adding or changing a particular data type
- [ ] [Function validation decorator](https://pydantic-docs.helpmanual.io/usage/validation_decorator/)
- [ ] [Generic Models](https://pydantic-docs.helpmanual.io/usage/models/#generic-models)
- [ ] [Other Model behaviour](https://pydantic-docs.helpmanual.io/usage/models/) - `construct()`, pickling, private attributes, ORM mode
- [X] [Plugins](https://pydantic-docs.helpmanual.io/) and integration with other tools - mypy, FastAPI, python-devtools, Hypothesis, VS Code, PyCharm, etc.

**Comment by samuelcolvin** [comment]
Awesome, looks interesting. Congratulations.

> i was wondering if there are plans to add a section for "built with pydantic" software like this and others?

Yes, absolutely would be good to add when we rebuild the docs for V2. Can go in the new docs repo once it's started, see https://github.com/pydantic/pydantic/issues/4684

**Reply by jhnnsrs** [reply]
Cool, that would be great. Happy to provide you with anything if i can be of help!

I saw the section about lighthouse projects to have test against. Turms is quite unknown at this point but would happily provide some tests if that would be of help!

Thanks for the great library. It is pure bliss to build upon!
